libs/frontend/ui-build/src/components/component-menu/ComponentInputs.vue

```diff
@@ -98,7 +98,9 @@ const inputArray = computed(() => {
     renderMode: RenderMode.Build,
     resolveTheme: false,
   })
-  return Object.entries(inputs)
+  return Object.entries(inputs).sort((entryA, entryB) =>
+    entryA[0].localeCompare(entryB[0]),
+  )
```

Comment:
The sorting was added to maintain the order of inputs while editing.

https://github.com/samatechtw/pubstudio-builder/assets/59676941/18444dc1-073f-406b-80c4-fe0488640eb5